ironfish/src/telemetry/telemetry.ts

```diff
@@ -281,9 +281,18 @@ export class Telemetry {
   }
 
   submitNodeStarted(): void {
+    const fields: Field[] = [{ name: 'online', type: 'boolean', value: true }]
+
+    if (this.metrics) {
+      fields.concat([
```

Comment:
You should double check what this method is doing to see if it does what you want.